packages/angular/src/generators/ng-add/utilities/workspace.ts

```diff
@@ -10,11 +10,13 @@ import {
   writeJson,
 } from '@nrwl/devkit';
 import { Linter, lintInitGenerator } from '@nrwl/linter';
-import { DEFAULT_NRWL_PRETTIER_CONFIG } from '@nrwl/workspace/src/generators/new/generate-workspace-files';
-import { deduceDefaultBase } from 'nx/src/utils/default-base';
-import { resolveUserExistingPrettierConfig } from '@nrwl/workspace/src/utilities/prettier';
 import { getRootTsConfigPathInTree } from '@nrwl/js';
-import { prettierVersion } from '@nrwl/workspace/src/utils/versions';
+import {
+  DEFAULT_NRWL_PRETTIER_CONFIG,
+  resolveUserExistingPrettierConfig,
+} from '@nrwl/js/src/utils/prettier';
+import { deduceDefaultBase } from 'nx/src/utils/default-base';
+import { prettierVersion } from '@nrwl/js/src/utils/versions';
```

Comment:
Don't worry too much about changing this to use `@nrwl/js:init` or something else because I think @leosvelperez is planning on removing this?

Reply:
Yeah, the change is alright, but it won't matter much as this will change.